### PR TITLE
DP-995 - Add translations to forms view models

### PR DESCRIPTION
**Pull request by rmohammed-goaco**
https://noticingsystem.atlassian.net/browse/DP-995

### CODE REVIEW

File: Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormsAnswerSetRemoveConfirmation.cshtml.cs [hunk 1]
```diff
@@ -24,7 +25,7 @@ public class FormsAnswerSetRemoveConfirmationModel(
     public Guid AnswerSetId { get; set; }
 
     [BindProperty]
-    [Required(ErrorMessage = "Please select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Global_SelectAnOption))]
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormsAnswerSetRemoveConfirmation.cshtml.cs:28]: Can we go with the standard `ErrorMessageResourceName` / `ErrorMessageResourceType` pairing please as found elsewhere. The plain `ErrorMessage` option is supposed to be reserved for untranslated text.

Comment (dbgoaco) [inline Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormsAnswerSetRemoveConfirmation.cshtml.cs:28]: I think the format needs to be:     `[Required(ErrorMessageResourceName = nameof(StaticTextResource.ApiKeyManagement_CreateApiKey_ValidationError), ErrorMessageResourceType = typeof(StaticTextResource))]`

File: Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormsCheckFurtherQuestionsExempted.cshtml.cs [hunk 2]
```diff
@@ -22,7 +23,7 @@ public class FormsCheckFurtherQuestionsExemptedModel(IFormsClient formsClient, I
 
 
     [BindProperty]
-    [Required(ErrorMessage = "Please select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Global_SelectAnOption))]
```
Comment (dbgoaco) [inline Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormsCheckFurtherQuestionsExempted.cshtml.cs:26]: Same here regarding parameters.

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormsAnswerSetRemoveConfirmation.cshtml.cs (hunk 1 above)
Comment (rmohammed-goaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormsAnswerSetRemoveConfirmation.cshtml.cs:28]: Ok. Thanks

Comment (rmohammed-goaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormsAnswerSetRemoveConfirmation.cshtml.cs:28]: Ok

**Review by andymantell — APPROVED** [review]

**Pull request merged**